packages/hardhat-core/test/internal/hardhat-network/helpers/providers.ts

```diff
@@ -194,3 +194,21 @@ if (INFURA_URL !== undefined) {
     },
   });
 }
+
+if (RETHNET_BINARY !== undefined) {
+  PROVIDERS.push({
+    name: "Rethnet",
+    isFork: false,
```

Comment:
@fvictorio should we add two prodivers? One with fork and one without?

Reply:
`isFork` means mainnet providers, so in the case of the rethnet provider is probably not useful. The rethnet provider is more akin to the Hardhat Node provider that is started in background.